Failures in [node-test-pull-request/69603](https://ci.nodejs.org/job/node-test-pull-request/69603/) to [node-test-pull-request/69701](https://ci.nodejs.org/job/node-test-pull-request/69701/) that failed 2 or more PRs
(Generated with `ncu-ci walk pr --stats=true --markdown reports/2025-10-13.md`)

| UTC Time         | RUNNING | SUCCESS | UNSTABLE | ABORTED | FAILURE | Green Rate |
| ---------------- | ------- | ------- | -------- | ------- | ------- | ---------- |
| 2025-10-13 00:11 | 1       | 1       | 31       | 2       | 65      | 1.03%      |


### Jenkins Failure

| Reason | <code>Build timed out (after 30 minutes). Marking the build as failed.</code> |
| - | :- |
| Type | JENKINS_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/60177/](https://ci.nodejs.org/job/node-test-pull-request/69666/), [https://github.com/nodejs/node/pull/60206/](https://ci.nodejs.org/job/node-test-pull-request/69667/)) |
| Appeared | [test-azure_msft-win11_vs2022-arm64-5](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=3,nodes=win11-arm64-COMPILED_BY-vs2022_clang-arm64/37212/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/69666/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/69667/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=3,nodes=win11-arm64-COMPILED_BY-vs2022_clang-arm64/37212/console">Example</a></summary>

```
Build timed out (after 30 minutes). Marking the build as failed.
```
</details>

-------


### Git Failure


### Build Failure


### JSTest Failure

| Reason | <code>parallel/test-without-async-context-frame</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 20 ([https://github.com/nodejs/node/pull/59889/](https://ci.nodejs.org/job/node-test-pull-request/69603/), [https://github.com/nodejs/node/pull/60153/](https://ci.nodejs.org/job/node-test-pull-request/69606/), [https://github.com/nodejs/node/pull/60160/](https://ci.nodejs.org/job/node-test-pull-request/69617/), [https://github.com/nodejs/node/pull/59853/](https://ci.nodejs.org/job/node-test-pull-request/69618/), [https://github.com/nodejs/node/pull/59931/](https://ci.nodejs.org/job/node-test-pull-request/69619/), [https://github.com/nodejs/node/pull/60148/](https://ci.nodejs.org/job/node-test-pull-request/69623/), [https://github.com/nodejs/node/pull/60144/](https://ci.nodejs.org/job/node-test-pull-request/69624/), [https://github.com/nodejs/node/pull/60174/](https://ci.nodejs.org/job/node-test-pull-request/69637/), [https://github.com/nodejs/node/pull/60156/](https://ci.nodejs.org/job/node-test-pull-request/69645/), [https://github.com/nodejs/node/pull/60168/](https://ci.nodejs.org/job/node-test-pull-request/69647/), [https://github.com/nodejs/node/pull/59807/](https://ci.nodejs.org/job/node-test-pull-request/69652/), [https://github.com/nodejs/node/pull/60062/](https://ci.nodejs.org/job/node-test-pull-request/69653/), [https://github.com/nodejs/node/pull/59983/](https://ci.nodejs.org/job/node-test-pull-request/69655/), [https://github.com/nodejs/node/pull/60133/](https://ci.nodejs.org/job/node-test-pull-request/69656/), [https://github.com/nodejs/node/pull/60131/](https://ci.nodejs.org/job/node-test-pull-request/69657/), [https://github.com/nodejs/node/pull/59873/](https://ci.nodejs.org/job/node-test-pull-request/69658/), [https://github.com/nodejs/node/pull/60126/](https://ci.nodejs.org/job/node-test-pull-request/69668/), [https://github.com/nodejs/node/pull/60177/](https://ci.nodejs.org/job/node-test-pull-request/69669/), [https://github.com/nodejs/node/pull/60175/](https://ci.nodejs.org/job/node-test-pull-request/69674/), [https://github.com/nodejs/node/pull/59876/](https://ci.nodejs.org/job/node-test-pull-request/69675/)) |
| Appeared | [vm-6ll6k](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67295/console), [vm-5tpep](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67294/console), [vm-c1xp1](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67289/console), [test-digitalocean-ubuntu2404_sharedlibs_container-x64-5](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2404_sharedlibs_shared_x64/53117/console), [vm-u76pj](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67278/console), [vm-bkqmy](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67275/console), [vm-g4tu2](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67273/console), [vm-12cnv](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67270/console), [vm-cc50q](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67269/console), [vm-0b80q](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67264/console), [test-ibm-ubuntu2404_sharedlibs_container-x64-3](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2404_sharedlibs_shared_x64/53105/console), [vm-hw69r](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67258/console), [vm-s4ci3](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67255/console), [vm-nd723](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67253/console), [vm-6zndc](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67236/console), [vm-hgw3t](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67235/console), [vm-ebad9](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67231/console), [vm-dm937](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67230/console), [vm-x4kfi](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67229/console), [vm-532du](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67215/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/69603/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/69675/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67295/console">Example</a></summary>

```
not ok 4191 parallel/test-without-async-context-frame
  ---
  duration_ms: 120146.07100
  severity: fail
  exitcode: -15
  stack: |-
    timeout
  ...

```
</details>

-------

| Reason | <code>sequential/test-worker-prof</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 18 ([https://github.com/nodejs/node/pull/60153/](https://ci.nodejs.org/job/node-test-pull-request/69606/), [https://github.com/nodejs/node/pull/59853/](https://ci.nodejs.org/job/node-test-pull-request/69614/), [https://github.com/nodejs/node/pull/60148/](https://ci.nodejs.org/job/node-test-pull-request/69623/), [https://github.com/nodejs/node/pull/60098/](https://ci.nodejs.org/job/node-test-pull-request/69638/), [https://github.com/nodejs/node/pull/60133/](https://ci.nodejs.org/job/node-test-pull-request/69639/), [https://github.com/nodejs/node/pull/60177/](https://ci.nodejs.org/job/node-test-pull-request/69642/), [https://github.com/nodejs/node/pull/60168/](https://ci.nodejs.org/job/node-test-pull-request/69644/), [https://github.com/nodejs/node/pull/60156/](https://ci.nodejs.org/job/node-test-pull-request/69645/), [https://github.com/nodejs/node/pull/60061/](https://ci.nodejs.org/job/node-test-pull-request/69648/), [https://github.com/nodejs/node/pull/60126/](https://ci.nodejs.org/job/node-test-pull-request/69660/), [https://github.com/nodejs/node/pull/59876/](https://ci.nodejs.org/job/node-test-pull-request/69675/), [https://github.com/nodejs/node/pull/60131/](https://ci.nodejs.org/job/node-test-pull-request/69676/), [https://github.com/nodejs/node/pull/60206/](https://ci.nodejs.org/job/node-test-pull-request/69689/), [https://github.com/nodejs/node/pull/59873/](https://ci.nodejs.org/job/node-test-pull-request/69690/), [https://github.com/nodejs/node/pull/60062/](https://ci.nodejs.org/job/node-test-pull-request/69691/), [https://github.com/nodejs/node/pull/60223/](https://ci.nodejs.org/job/node-test-pull-request/69694/), [https://github.com/nodejs/node/pull/60225/](https://ci.nodejs.org/job/node-test-pull-request/69696/), [https://github.com/nodejs/node/pull/60217/](https://ci.nodejs.org/job/node-test-pull-request/69701/)) |
| Appeared | [test-azure_msft-win10_vs2019-x64-2](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win10-COMPILED_BY-vs2022_clang/37242/console), [test-azure_msft-win11_vs2022-arm64-5](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=3,nodes=win11-arm64-COMPILED_BY-vs2022_clang-arm64/37242/console), [test-azure_msft-win11_vs2022-x64-2](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win11-COMPILED_BY-vs2022_clang/37227/console), [test-azure_msft-win10_vs2019-x64-3](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win10-COMPILED_BY-vs2022_clang/37229/console), [test-azure_msft-win11_vs2022-x64-3](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win11-COMPILED_BY-vs2022_clang/37214/console), [test-azure_msft-win11_vs2022-arm64-4](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=1,nodes=win11-arm64-COMPILED_BY-vs2022_clang-arm64/37215/console), [test-azure_msft-win11_vs2022-arm64-2](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=1,nodes=win11-arm64-COMPILED_BY-vs2022_clang-arm64/37199/console), [test-azure_msft-win10_vs2019-x64-4](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win10-COMPILED_BY-vs2022_clang/37189/console), [test-azure_msft-win11_vs2022-x64-1](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win11-COMPILED_BY-vs2022_clang/37191/console), [test-ibm-rhel8-s390x-3](https://ci.nodejs.org/job/node-test-commit-linuxone/nodes=rhel8-s390x/51901/console), [test-linuxonecc-rhel9-s390x-1](https://ci.nodejs.org/job/node-test-commit-linuxone/nodes=rhel9-s390x/51898/console), [test-azure_msft-win11_vs2022-arm64-6](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win11-arm64-COMPILED_BY-vs2022_clang-arm64/37173/console), [test-ibm-rhel8-s390x-1](https://ci.nodejs.org/job/node-test-commit-linuxone/nodes=rhel8-s390x/51877/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/69606/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/69701/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win10-COMPILED_BY-vs2022_clang/37242/console">Example</a></summary>

```
not ok 1144 sequential/test-worker-prof
  ---
  duration_ms: 2383.03100
  severity: fail
  exitcode: 1
  stack: |-
    files found in working directory before worker starts [ 'isolate-00000271CFEE5000-4288-v8.log' ]
    parent prof file: isolate-00000271CFEE5000-4288-v8.log
    parent posted 46869 messages
    files found in working directory before worker exits [
      'isolate-00000271CFEE5000-4288-v8.log',
      'isolate-00000271E94C7000-4288-v8.log'
    ]
    worker prof file: isolate-00000271E94C7000-4288-v8.log
    
    worker ticks 33
    parent ticks 15
    node:internal/assert/utils:77
        throw err;
        ^
    
    AssertionError [ERR_ASSERTION]: parent ticks <= 15:
    tick,0x7ff73a3c0d87,110658,0,0x0,5,0x2bc3f95907d,0x2bc3f959bcb,0x35df1787094
    tick,0x7ff73a5567b0,190565,0,0x0,2,0x7ff739fb0fc0,0x2719fe7b4fa,0x35df179b9ca,0x35df179b7a5,0x35df1799ac8,0x1057733e7df,0x1057733efaf,0x35df178c9d0,0x35df178c890,0x2718004058f,0x27180040287,0x35df178c5ed,0x2718004058f,0x27180040287,0x35df178bde7,0...
```
</details>

-------

| Reason | <code>parallel/test-http2-close-while-writing</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 13 ([https://github.com/nodejs/node/pull/60174/](https://ci.nodejs.org/job/node-test-pull-request/69637/), [https://github.com/nodejs/node/pull/60098/](https://ci.nodejs.org/job/node-test-pull-request/69638/), [https://github.com/nodejs/node/pull/60133/](https://ci.nodejs.org/job/node-test-pull-request/69639/), [https://github.com/nodejs/node/pull/60131/](https://ci.nodejs.org/job/node-test-pull-request/69640/), [https://github.com/nodejs/node/pull/60177/](https://ci.nodejs.org/job/node-test-pull-request/69642/), [https://github.com/nodejs/node/pull/60168/](https://ci.nodejs.org/job/node-test-pull-request/69647/), [https://github.com/nodejs/node/pull/60061/](https://ci.nodejs.org/job/node-test-pull-request/69650/), [https://github.com/nodejs/node/pull/60156/](https://ci.nodejs.org/job/node-test-pull-request/69651/), [https://github.com/nodejs/node/pull/59807/](https://ci.nodejs.org/job/node-test-pull-request/69652/), [https://github.com/nodejs/node/pull/60062/](https://ci.nodejs.org/job/node-test-pull-request/69653/), [https://github.com/nodejs/node/pull/60206/](https://ci.nodejs.org/job/node-test-pull-request/69667/), [https://github.com/nodejs/node/pull/60175/](https://ci.nodejs.org/job/node-test-pull-request/69674/), [https://github.com/nodejs/node/pull/60199/](https://ci.nodejs.org/job/node-test-pull-request/69682/)) |
| Appeared | [vm-yx2lw](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67304/console), [vm-5tpep](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67294/console), [vm-r3u8q](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67287/console), [vm-xc3hr](https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx13-x64/67252/console), [vm-n9jtk](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67266/console), [vm-nd723](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67265/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/69637/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/69682/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67304/console">Example</a></summary>

```
not ok 2461 parallel/test-http2-close-while-writing
  ---
  duration_ms: 120146.73500
  severity: fail
  exitcode: -15
  stack: |-
    timeout
  ...

```
</details>

-------

| Reason | <code>parallel/test-runner-watch-mode-kill-signal</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 11 ([https://github.com/nodejs/node/pull/59432/](https://ci.nodejs.org/job/node-test-pull-request/69611/), [https://github.com/nodejs/node/pull/60148/](https://ci.nodejs.org/job/node-test-pull-request/69615/), [https://github.com/nodejs/node/pull/60156/](https://ci.nodejs.org/job/node-test-pull-request/69651/), [https://github.com/nodejs/node/pull/59807/](https://ci.nodejs.org/job/node-test-pull-request/69652/), [https://github.com/nodejs/node/pull/59983/](https://ci.nodejs.org/job/node-test-pull-request/69655/), [https://github.com/nodejs/node/pull/60133/](https://ci.nodejs.org/job/node-test-pull-request/69656/), [https://github.com/nodejs/node/pull/60177/](https://ci.nodejs.org/job/node-test-pull-request/69669/), [https://github.com/nodejs/node/pull/60175/](https://ci.nodejs.org/job/node-test-pull-request/69674/), [https://github.com/nodejs/node/pull/60168/](https://ci.nodejs.org/job/node-test-pull-request/69681/), [https://github.com/nodejs/node/pull/60199/](https://ci.nodejs.org/job/node-test-pull-request/69682/), [https://github.com/nodejs/node/pull/60062/](https://ci.nodejs.org/job/node-test-pull-request/69698/)) |
| Appeared | [vm-x37tr](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67326/console), [vm-yx2lw](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67304/console), [vm-i57a3](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67303/console), [vm-5tpep](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67294/console), [vm-c1xp1](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67289/console), [vm-u76pj](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67274/console), [vm-g4tu2](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67273/console), [vm-cc50q](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67269/console), [vm-hr6tf](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67268/console), [vm-n9jtk](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67266/console), [vm-ebad9](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67227/console), [vm-htd7e](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-arm64/67222/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/69611/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/69698/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67326/console">Example</a></summary>

```
not ok 3723 parallel/test-runner-watch-mode-kill-signal
  ---
  duration_ms: 120074.67600
  severity: fail
  exitcode: -15
  stack: |-
    timeout
  ...

```
</details>

-------

| Reason | <code>sequential/test-perf-hooks</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 11 ([https://github.com/nodejs/node/pull/60153/](https://ci.nodejs.org/job/node-test-pull-request/69606/), [https://github.com/nodejs/node/pull/60174/](https://ci.nodejs.org/job/node-test-pull-request/69637/), [https://github.com/nodejs/node/pull/60098/](https://ci.nodejs.org/job/node-test-pull-request/69638/), [https://github.com/nodejs/node/pull/60133/](https://ci.nodejs.org/job/node-test-pull-request/69639/), [https://github.com/nodejs/node/pull/60177/](https://ci.nodejs.org/job/node-test-pull-request/69642/), [https://github.com/nodejs/node/pull/60168/](https://ci.nodejs.org/job/node-test-pull-request/69647/), [https://github.com/nodejs/node/pull/60061/](https://ci.nodejs.org/job/node-test-pull-request/69650/), [https://github.com/nodejs/node/pull/60156/](https://ci.nodejs.org/job/node-test-pull-request/69651/), [https://github.com/nodejs/node/pull/59807/](https://ci.nodejs.org/job/node-test-pull-request/69652/), [https://github.com/nodejs/node/pull/60062/](https://ci.nodejs.org/job/node-test-pull-request/69653/), [https://github.com/nodejs/node/pull/60131/](https://ci.nodejs.org/job/node-test-pull-request/69688/)) |
| Appeared | [vm-1uzqj](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67313/console), [vm-bkqmy](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67275/console), [vm-xc3hr](https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx13-x64/67252/console), [vm-nd723](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67265/console), [test-digitalocean-ubuntu2404_sharedlibs_container-x64-9](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2404_sharedlibs_openssl31_x64/53105/console), [vm-s4ci3](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67255/console), [vm-532du](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67215/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/69606/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/69688/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67313/console">Example</a></summary>

```
not ok 4762 sequential/test-perf-hooks
  ---
  duration_ms: 1190.35600
  severity: fail
  exitcode: 1
  stack: |-
    node:internal/assert/utils:77
        throw err;
        ^
    
    AssertionError [ERR_ASSERTION]: Date.now() - performance.timeOrigin (603.803955078125) - process.uptime() * 1000 (437.110328) = 166.69362707812502 >= +- 50
        at Object.<anonymous> (/Users/admin/build/workspace/node-test-commit-osx/nodes/macos15-x64/test/sequential/test-perf-hooks.js:36:3)
        at Module._compile (node:internal/modules/cjs/loader:1759:14)
        at Object..js (node:internal/modules/cjs/loader:1892:10)
        at Module.load (node:internal/modules/cjs/loader:1479:32)
        at Module._load (node:internal/modules/cjs/loader:1298:12)
        at TracingChannel.traceSync (node:diagnostics_channel:328:14)
        at wrapModuleLoad (node:internal/modules/cjs/loader:244:24)
        at Module.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:154:5)
        at node:internal/main/run_main_modul...
```
</details>

-------

| Reason | <code>client-proxy/test-https-proxy-request-invalid-char-in-url</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 5 ([https://github.com/nodejs/node/pull/59432/](https://ci.nodejs.org/job/node-test-pull-request/69612/), [https://github.com/nodejs/node/pull/59853/](https://ci.nodejs.org/job/node-test-pull-request/69618/), [https://github.com/nodejs/node/pull/60098/](https://ci.nodejs.org/job/node-test-pull-request/69638/), [https://github.com/nodejs/node/pull/60061/](https://ci.nodejs.org/job/node-test-pull-request/69648/), [https://github.com/nodejs/node/pull/59807/](https://ci.nodejs.org/job/node-test-pull-request/69649/)) |
| Appeared | [vm-n9jtk](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67266/console), [vm-nd723](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67265/console), [vm-kxsdk](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67254/console), [vm-dm937](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67230/console), [vm-qghf4](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67223/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/69612/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/69649/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67266/console">Example</a></summary>

```
not ok 4376 client-proxy/test-https-proxy-request-invalid-char-in-url
  ---
  duration_ms: 761.56900
  severity: fail
  exitcode: 1
  stack: |-
    [Upstream server] responding to request for '/newline-in-host'
    [Upstream server] responding to request for '/carriage-return-in-host'
    [Upstream server] responding to request for '/crlf-in-host'
    [Proxy client] Received response from server for 'https://local\nhost:55432/newline-in-host': Response for '/newline-in-host'
    [Proxy client] #1 closed request for: 'https://local\nhost:55432/newline-in-host'
    [Proxy client] Received response from server for 'https://local\rhost:55432/carriage-return-in-host': Response for '/carriage-return-in-host'
    [Proxy client] #2 closed request for: 'https://local\rhost:55432/carriage-return-in-host'
    [Proxy client] Received response from server for 'https://local\r\nhost:55432/crlf-in-host': Response for '/crlf-in-host'
    [Proxy client] #3 closed request for: 'https://local\r\nhost:55432/crlf-in-host'
    [Up...
```
</details>

-------

| Reason | <code>parallel/test-quic-test-client</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 5 ([https://github.com/nodejs/node/pull/60168/](https://ci.nodejs.org/job/node-test-pull-request/69628/), [https://github.com/nodejs/node/pull/60061/](https://ci.nodejs.org/job/node-test-pull-request/69650/), [https://github.com/nodejs/node/pull/60174/](https://ci.nodejs.org/job/node-test-pull-request/69662/), [https://github.com/nodejs/node/pull/60175/](https://ci.nodejs.org/job/node-test-pull-request/69674/), [https://github.com/nodejs/node/pull/60206/](https://ci.nodejs.org/job/node-test-pull-request/69677/)) |
| Appeared | [vm-5tpep](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67297/console), [vm-rpw04](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67282/console), [vm-n9jtk](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67267/console), [vm-ibfq1](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67250/console), [vm-nvuji](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67240/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/69628/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/69677/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67297/console">Example</a></summary>

```
not ok 2573 parallel/test-quic-test-client
  ---
  duration_ms: 1509.45700
  severity: fail
  exitcode: 1
  stack: |-
    /Users/admin/build/workspace/node-test-commit-osx/nodes/macos15-x64/out/Release/ngtcp2_test_client
    node:internal/modules/run_main:107
        triggerUncaughtException(
        ^
    
    AssertionError [ERR_ASSERTION]: Missing expected rejection.
        at process.processTicksAndRejections (node:internal/process/task_queues:103:5)
        at async file:///Users/admin/build/workspace/node-test-commit-osx/nodes/macos15-x64/test/parallel/test-quic-test-client.mjs:36:1 {
      generatedMessage: false,
      code: 'ERR_ASSERTION',
      actual: undefined,
      expected: { message: /Process exited with code 1 and signal null/ },
      operator: 'rejects',
      diff: 'simple'
    }
    
    Node.js v25.0.0-pre
  ...

```
</details>

-------

| Reason | <code>wpt/test-hr-time</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 5 ([https://github.com/nodejs/node/pull/60160/](https://ci.nodejs.org/job/node-test-pull-request/69617/), [https://github.com/nodejs/node/pull/59853/](https://ci.nodejs.org/job/node-test-pull-request/69618/), [https://github.com/nodejs/node/pull/60174/](https://ci.nodejs.org/job/node-test-pull-request/69637/), [https://github.com/nodejs/node/pull/60131/](https://ci.nodejs.org/job/node-test-pull-request/69640/), [https://github.com/nodejs/node/pull/60061/](https://ci.nodejs.org/job/node-test-pull-request/69650/)) |
| Appeared | [vm-n9jtk](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67267/console), [vm-nd723](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67256/console), [vm-dm937](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67230/console), [vm-x4kfi](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67229/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/69617/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/69650/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67267/console">Example</a></summary>

```
not ok 4821 wpt/test-hr-time
  ---
  duration_ms: 2399.56500
  severity: fail
  exitcode: 1
  stack: |-
    [SKIPPED] idlharness-shadowrealm.window.js: ShadowRealm support is not enabled
    [SKIPPED] window-worker-timeOrigin.window.js: depends on URL.createObjectURL(blob)
    [PASS] self.performance.now() is a function that returns a number
    [PASS] self.performance.now() returns a positive number
    [PASS] self.performance.now() difference is not negative
    [PASS] Performance interface extends EventTarget.
    [PASS] self.performance.now() returns a positive number
    [PASS] self.performance.now() difference is not negative
    [PASS] idl_test validation
    [PASS] Partial interface mixin WindowOrWorkerGlobalScope: original interface mixin defined
    [PASS] Partial interface mixin WindowOrWorkerGlobalScope: member names are unique
    [PASS] Partial interface Window: member names are unique
    [PASS] Window includes GlobalEventHandlers: member names are unique
    [PASS] Window includes WindowEventH...
```
</details>

-------

| Reason | <code>parallel/test-runner-output</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 4 ([https://github.com/nodejs/node/pull/59396/](https://ci.nodejs.org/job/node-test-pull-request/69643/), [https://github.com/nodejs/node/pull/59983/](https://ci.nodejs.org/job/node-test-pull-request/69655/), [https://github.com/nodejs/node/pull/60208/](https://ci.nodejs.org/job/node-test-pull-request/69679/), [https://github.com/nodejs/node/pull/60131/](https://ci.nodejs.org/job/node-test-pull-request/69688/)) |
| Appeared | [vm-ptrav](https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx13-x64/67307/console), [undefined](https://ci.nodejs.org/job/node-test-commit-custom-suites-freestyle/44363/console), [vm-g4tu2](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67273/console), [test-ibm-ubuntu2404-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=ubuntu2404-x64/66934/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/69643/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/69688/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx13-x64/67307/console">Example</a></summary>

```
not ok 3868 parallel/test-runner-output
  ---
  duration_ms: 120117.61600
  severity: fail
  exitcode: -15
  stack: |-
    timeout
    TAP version 13
    # Subtest: test runner output
        # Subtest: test-runner/output/abort.js
        ok 1 - test-runner/output/abort.js
          ---
          duration_ms: 1438.369832
          ...

```
</details>

-------

| Reason | <code>parallel/test-cluster-fork-stdio</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/59396/](https://ci.nodejs.org/job/node-test-pull-request/69643/), [https://github.com/nodejs/node/pull/60131/](https://ci.nodejs.org/job/node-test-pull-request/69672/)) |
| Appeared | [test-rackspace-win2019_vs2019-x64-2](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win2019-COMPILED_BY-vs2022_clang/37213/console), [test-rackspace-win2019_vs2019-x64-3](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win2019-COMPILED_BY-vs2022_clang/37192/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/69643/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/69672/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win2019-COMPILED_BY-vs2022_clang/37213/console">Example</a></summary>

```
not ok 1097 parallel/test-cluster-fork-stdio
  ---
  duration_ms: 120191.26200
  severity: fail
  exitcode: 1
  stack: |-
    timeout
  ...

```
</details>

-------

| Reason | <code>parallel/test-zlib-brotli-16GB</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/60153/](https://ci.nodejs.org/job/node-test-pull-request/69606/), [https://github.com/nodejs/node/pull/60148/](https://ci.nodejs.org/job/node-test-pull-request/69615/)) |
| Appeared | [vm-ebad9](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67227/console), [vm-532du](https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67215/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/69606/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/69615/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-osx/nodes=macos15-x64/67227/console">Example</a></summary>

```
not ok 3857 parallel/test-zlib-brotli-16GB
  ---
  duration_ms: 1422.15600
  severity: fail
  exitcode: 1
  stack: |-
    node:assert:149
      throw new AssertionError(obj);
      ^
    
    AssertionError [ERR_ASSERTION]: Expected values to be strictly equal:
    
    0 !== 4
    
        at Timeout.<anonymous> (/Users/admin/build/workspace/node-test-commit-osx/nodes/macos15-x64/test/parallel/test-zlib-brotli-16GB.js:22:3)
        at Timeout._onTimeout (/Users/admin/build/workspace/node-test-commit-osx/nodes/macos15-x64/test/common/index.js:475:15)
        at listOnTimeout (node:internal/timers:605:17)
        at process.processTimers (node:internal/timers:541:7) {
      generatedMessage: true,
      code: 'ERR_ASSERTION',
      actual: 0,
      expected: 4,
      operator: 'strictEqual',
      diff: 'simple'
    }
    
    Node.js v25.0.0-pre
  ...

```
</details>

-------

| Reason | <code>wpt/test-wasm-webapi</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/60148/](https://ci.nodejs.org/job/node-test-pull-request/69615/), [https://github.com/nodejs/node/pull/60177/](https://ci.nodejs.org/job/node-test-pull-request/69642/)) |
| Appeared | [test-azure_msft-win11_vs2022-arm64-2](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win11-arm64-COMPILED_BY-vs2022_clang-arm64/37191/console), [test-azure_msft-win11_vs2022-arm64-4](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=1,nodes=win11-arm64-COMPILED_BY-vs2022_clang-arm64/37164/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/69615/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/69642/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win11-arm64-COMPILED_BY-vs2022_clang-arm64/37191/console">Example</a></summary>

```
not ok 1148 wpt/test-wasm-webapi
  ---
  duration_ms: 432.68200
  severity: crashed
  exitcode: 3221225477
  stack: |-
    [SKIPPED] abort.any.js: WPTRunner does not support fetch()
    [SKIPPED] contenttype.any.js: WPTRunner does not support fetch()
    [SKIPPED] historical.any.js: indexedDB is not defined
    [SKIPPED] idlharness.any.js: not configured
    [SKIPPED] instantiateStreaming-bad-imports.any.js: Flaky on ARM with V8 >= 11.2
    [SKIPPED] origin.sub.any.js: CORS not implemented
    [SKIPPED] status.any.js: WPTRunner does not support fetch()
    [PASS] compileStreaming
    [PASS] instantiateStreaming
    [PASS] compileStreaming: no body
    [PASS] compileStreaming: no body in a promise
    [PASS] compileStreaming: undefined
    [PASS] compileStreaming: undefined in a promise
    [PASS] compileStreaming: null
    [PASS] compileStreaming: null in a promise
    [PASS] compileStreaming: true
    [PASS] compileStreaming: true in a promise
    [PASS] compileStreaming: "test"
    [PASS] compileStreaming: ...
```
</details>

-------

